### PR TITLE
ONBOARDING: Calculations completed

### PR DESCRIPTION
**Pull request by priettt**
Base operators working. Results are showed and can be used for further calculations

### CODE REVIEW

File: app/src/main/java/com/globant/franciscoprieto/kotlincalculator/mvp/presenter/CalculatorPresenter.kt [hunk 1]
```diff
@@ -1,36 +1,82 @@
 package com.globant.franciscoprieto.kotlincalculator.mvp.presenter
```
Comment (fanky10) [inline app/src/main/java/com/globant/franciscoprieto/kotlincalculator/mvp/presenter/CalculatorPresenter.kt:1]: the lack of unit tests is disturbing, please add.

File: app/src/main/java/com/globant/franciscoprieto/kotlincalculator/mvp/model/CalculatorModel.kt [hunk 2]
```diff
@@ -2,9 +2,16 @@ package com.globant.franciscoprieto.kotlincalculator.mvp.model
 
 class CalculatorModel {
```
Comment (fanky10) [inline app/src/main/java/com/globant/franciscoprieto/kotlincalculator/mvp/model/CalculatorModel.kt:3]: add unit tests here as well

**Review by fanky10 — CHANGES_REQUESTED** [review]
check comments

File: app/src/main/java/com/globant/franciscoprieto/kotlincalculator/mvp/presenter/CalculatorPresenter.kt (hunk 1 above)
Comment (priettt) [reply app/src/main/java/com/globant/franciscoprieto/kotlincalculator/mvp/presenter/CalculatorPresenter.kt:1]: done!

File: app/src/main/java/com/globant/franciscoprieto/kotlincalculator/mvp/model/CalculatorModel.kt (hunk 2 above)
Comment (priettt) [reply app/src/main/java/com/globant/franciscoprieto/kotlincalculator/mvp/model/CalculatorModel.kt:3]: done!

**Review by fanky10 — APPROVED** [review]
LG2M

**Comment by SartiMau** [comment]
👍

**Review by SartiMau — APPROVED** [review]

**Pull request merged**